eos/rare-b-decays/observables.cc

```diff
@@ -334,7 +335,11 @@ namespace eos
 
                 /// Branching ratio of B -> K^* psi
                 make_observable("B->K^*charmonium::branching_ratio",
-                        &BToKstarCharmonium::branching_ratio)
+                        &BToKstarCharmonium::branching_ratio),
+
+                // Branching ratio of B -> phi psi
+                make_observable("B_s->phicharmonium::branching_ratio",
```

Comment:
Argh, I overlooked this in a previous commit: Please use the following notation for the observables, to make it more manageable:
```
B->K^*psi::BR
B_s->phi::psi::BR
```
etc.

Reply:
Why do you have :: between phi and psi and not between K^* and psi ?

Reply:
The idea is to use the name of the process as a namespace / prefix.

``B->K^*psi::BR`` is the observable ``BR`` in the ``B->K^*psi`` namespace.

Reply:
Then I would write `B_s->phipsi::BR`, or am I missing something?

Reply:
Yes. Ah, I mistyped the original comment.  How did that happen?  ```B_s->phipsi::BR``` is what I'm after.

Reply:
Excellent, I will do the change, thanks!